app/components/about_page/belief_component.html.erb

```diff
@@ -1,13 +1,15 @@
 <div class="sm:grid sm:grid-cols-12 sm:gap-x-6 pb-14">
-  <div class="flex sm:col-span-2 justify-center ">
-    <%= image_tag @belief[:image_path], alt: '', class: 'w-20 h-20' %>
+  <div class="flex sm:col-span-2 justify-center">
+    <div class="w-20 h-20 flex items-center justify-center rounded-full bg-gray-700 dark:bg-white shadow-md">
```

Comment:
The backgrounds being different from the surrounding background is drawing the eye a bit too much. As a test - try reading the belief content and see where your eyes want to go to. I think this will distract users from the main content.

What did issues did you find with the contrast of these icons? They appear fine to me and lighthouse isn't raising anything about em.